### DISCUSSION TITLE
Memory problem with scipy and fft

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#300] [Q&A]
Hi, when I use MSNoise, I can see some errors related to memory occurred by sf.fftn or sf.ifft etc and stopped.
Calculation of fft related things can be calculated with numpy and scipy, and when I did quick test, using numpy is faster (almost 2~3 times).
When I see the code, you used numpy for other parts of the fft calculation except fft itself.
So I wonder why you used scipy for calculation of fft.
Is there a reason that I did not recognize? If there is, I have to know to solve memory problem.

And cc computation is proceeded with 0.6~0.8 %Memory, but it sometimes goes up almost 6.8%.
I have no idea to solve it.

The error message is as below even though memory is enough. (related to sf.fftn, sf.ifft, sf.fftfreq in s03compute_no_rotation.py)
`numpy.core._exceptions._ArrayMemoryError: Unable to allocate 84.6 MiB for an array with shape (140, 79233) and data type float64`

p.s. even numpy.fft also shows memory problem with the same code. Do you have any idea for this problem ?

**Comment by ThomasLecocq** [comment]
dupplicate of #301 , closing;
